bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java

```diff
@@ -228,52 +150,50 @@ public BatchExecutor(
    * @param <T> The type of the callback.
    * @return A ListenableFuture that will have the result when the RPC completes.
    */
-  private <R extends Row, T> ListenableFuture<Object> issueRowRequest(final Row row,
-      final Batch.Callback<T> callback, final Object[] results, final int index)
-      throws InterruptedException {
+  @SuppressWarnings("unchecked")
+  private <R extends Row, T> ListenableFuture<Result> issueAsyncRowRequest(Row row,
+      Batch.Callback<T> callback, Object[] results, int index) throws InterruptedException {
```

Comment:
Are we expecting caller to get result from ListenableFuture<Result> or "Object[] results, int index" ?